src/components/activity-header/header.scss

```diff
@@ -7,6 +7,10 @@
   font-size: 18px;
   font-weight: bold;
 
+  &.no-margin {
```

Comment:
It seems like `no-margin` is only used for the activity header when a sequence name is shown. If that's true, would it be slightly better to use a class name that's more indicative of that? Something like `show-sequence` or `in-sequence`? Or do you foresee this class possibly being used for other purposes?

Reply:
Good call.  More semantic this way. Fixed.